reactioncommerce/reaction
# refactor: use npm package for taxes plugin

Resolves #6182 

This PR removes the internal `taxes` plugin, and installs the new `api-plugin-taxes` from npm as a replacement.

The code in that npm plugin is the same, so the expected outcome of this PR is seeing nothing different than how the app currently runs.

## Notes

Since this is the final package being moved, there are a few additional changes that have been made in this PR that weren't made in the other "move plugin" PRs:

- This PR re-orders the `plugin.json` file (and renames a few of the keys) to be in the same order as we loaded the plugins while using `registerPlugins.js`.

- This PR removes the `core-services` and `plugins` folders, as there is no longer anything inside of these folders.

- This PR removes the `test:unit` scripts from `package.json`, as all unit tests have now been moved into plugins

- This PR removes the unit test steps from CI

## Testing
- Start reaction, see that `taxes` is the correct version in the `systemInformation` query
- Add a tax code, update taxes on a product
